Error while trying to start overfast-api

**Describe the bug**
Errored when trying to run overfast-api in production

**To Reproduce**
1. `git clone https://github.com/TeKrop/overfast-api && cd overfast-api`
2. Created and edited `.env` and changed `APP_PORT`, `APP_BASE_URL`
3. `apt install make` (GNU make 4.3)
4. make up (inside folder `overfast-api`)
5. shows error
```
root@itzdrli-dev-armv8:/home/dev/overfast-api# make up
Building OverFastAPI (production mode)...
docker compose build
WARN[0000] /home/dev/overfast-api/docker-compose.yml: `version` is obsolete 
[+] Building 43.4s (37/44)                                                                                                                                                                            docker:default
 => [redis internal] load build definition from Dockerfile                                                                                                                                                      0.0s
 => => transferring dockerfile: 109B                                                                                                                                                                            0.0s
 => [redis internal] load metadata for docker.io/library/redis:alpine                                                                                                                                           1.2s
 => [redis internal] load .dockerignore                                                                                                                                                                         0.0s
 => => transferring context: 2B                                                                                                                                                                                 0.0s
 => [redis 1/4] FROM docker.io/library/redis:alpine@sha256:0389bb8416d7c6ed065c25745179bf5d358e5d9472dd30a687ab36ffbb650262                                                                                     1.3s
 => => resolve docker.io/library/redis:alpine@sha256:0389bb8416d7c6ed065c25745179bf5d358e5d9472dd30a687ab36ffbb650262                                                                                           0.0s
 => => sha256:0389bb8416d7c6ed065c25745179bf5d358e5d9472dd30a687ab36ffbb650262 10.20kB / 10.20kB                                                                                                                0.0s
 => => sha256:94747bd812346354fd5042870b6e44e5406880a4e6b5736a9cf9c753110df11b 4.09MB / 4.09MB                                                                                                                  0.3s
 => => sha256:cdfbf2aae844ce899e13c8e08b524cfba67d9ddd6c5ca10be36c85aac3dc2ce7 961B / 961B                                                                                                                      0.1s
 => => sha256:16098e28f60cd9af53fb043cf88a98958b3760bc0ac96b7a4e8ee40ce1e3e7d3 2.48kB / 2.48kB                                                                                                                  0.0s
 => => sha256:cdb453bbf44653b2fff791f1950bc858a4156551f0a99eb868a81e17144a5297 7.92kB / 7.92kB                                                                                                                  0.0s
 => => sha256:510e151e1774050d5f5bb60a85736a80fa6c006791db5cebe6d506d65dec84a1 178.15kB / 178.15kB                                                                                                              0.3s
 => => sha256:70498b03edf90991b22ce539adf5694ff45163da2c71ec1501e5611fd228913e 934.59kB / 934.59kB                                                                                                              0.4s
 => => extracting sha256:94747bd812346354fd5042870b6e44e5406880a4e6b5736a9cf9c753110df11b                                                                                                                       0.2s
 => => sha256:b4a58c1d515ac6de0babaf7b798a89ae1a6707d9b0a70b647b735060b7da96cd 12.14MB / 12.14MB                                                                                                                0.5s
 => => sha256:f5bb1f316352fd460172c7e484e857550bb23940c213e90eeda6853f5c7f80dc 97B / 97B                                                                                                                        0.5s
 => => extracting sha256:cdfbf2aae844ce899e13c8e08b524cfba67d9ddd6c5ca10be36c85aac3dc2ce7                                                                                                                       0.0s
 => => sha256:8e79f523a11aead29e92102b8b50db1bca63e4571c30cda9b899fdee8177ed98 573B / 573B                                                                                                                      0.6s
 => => sha256:4f4fb700ef54461cfa02571ae0db9a0dc1e0cdb5577484a6d75e68dc38e8acc1 32B / 32B                                                                                                                        0.5s
 => => extracting sha256:510e151e1774050d5f5bb60a85736a80fa6c006791db5cebe6d506d65dec84a1                                                                                                                       0.1s
 => => extracting sha256:70498b03edf90991b22ce539adf5694ff45163da2c71ec1501e5611fd228913e                                                                                                                       0.1s
 => => extracting sha256:b4a58c1d515ac6de0babaf7b798a89ae1a6707d9b0a70b647b735060b7da96cd                                                                                                                       0.5s
 => => extracting sha256:f5bb1f316352fd460172c7e484e857550bb23940c213e90eeda6853f5c7f80dc                                                                                                                       0.0s
 => => extracting sha256:4f4fb700ef54461cfa02571ae0db9a0dc1e0cdb5577484a6d75e68dc38e8acc1                                                                                                                       0.0s
 => => extracting sha256:8e79f523a11aead29e92102b8b50db1bca63e4571c30cda9b899fdee8177ed98                                                                                                                       0.0s
 => [redis internal] load build context                                                                                                                                                                         0.0s
 => => transferring context: 878B                                                                                                                                                                               0.0s
 => [redis 2/4] WORKDIR /redis                                                                                                                                                                                  0.1s
 => [redis 3/4] COPY init.sh ./                                                                                                                                                                                 0.0s
 => [redis 4/4] RUN chmod +x init.sh                                                                                                                                                                            0.2s
 => [redis] exporting to image                                                                                                                                                                                  0.0s
 => => exporting layers                                                                                                                                                                                         0.0s
 => => writing image sha256:2c92f9ca2855eb09cbfdea5edabec9d27a207e0ca51a6d7dd5a51a1263b2e513                                                                                                                    0.0s
 => => naming to docker.io/library/overfast-api-redis                                                                                                                                                           0.0s
 => [app internal] load build definition from Dockerfile                                                                                                                                                        0.0s
 => => transferring dockerfile: 1.04kB                                                                                                                                                                          0.0s
 => [worker internal] load metadata for docker.io/library/python:3.12-alpine                                                                                                                                    1.2s
 => [worker internal] load build definition from Dockerfile                                                                                                                                                     0.0s
 => => transferring dockerfile: 1.04kB                                                                                                                                                                          0.0s
 => [app internal] load .dockerignore                                                                                                                                                                           0.0s
 => => transferring context: 2B                                                                                                                                                                                 0.0s
 => [worker internal] load .dockerignore                                                                                                                                                                        0.0s
 => => transferring context: 2B                                                                                                                                                                                 0.0s
 => [app main  1/10] FROM docker.io/library/python:3.12-alpine@sha256:d24ed567ee3b972478a232ceff84b0d002e18ee9f5d38234ecbffece23dfa084                                                                          1.8s
 => => resolve docker.io/library/python:3.12-alpine@sha256:d24ed567ee3b972478a232ceff84b0d002e18ee9f5d38234ecbffece23dfa084                                                                                     0.0s
 => => sha256:d24ed567ee3b972478a232ceff84b0d002e18ee9f5d38234ecbffece23dfa084 1.65kB / 1.65kB                                                                                                                  0.0s
 => => sha256:6714e211c290dc39908616ed0befdcc8f9097b69ee62d17f2f1833d764b93ef0 1.37kB / 1.37kB                                                                                                                  0.0s
 => => sha256:0fe205a492b75b71368b95db211dc148d50240d50752f04b34d1024084561296 6.64kB / 6.64kB                                                                                                                  0.0s
 => => sha256:820650aa2f5b8e795b04058fa617fb87ad3397f0fc2582400f9855b84f2d4401 465.47kB / 465.47kB                                                                                                              0.4s
 => => sha256:4e0f3549117ae48d35d3cbea60bca5ef0d654723e8065274c93137c3f2b44a60 14.68MB / 14.68MB                                                                                                                0.4s
 => => sha256:a5b6895e2863b5497adb5d68fe8e5241835c9624c9a31fd2b8532b08da56b638 242B / 242B                                                                                                                      0.1s
 => => sha256:629a421706977cd903f62bb22483a0fa6cba236a94d23543e616bf5c24516467 2.74MB / 2.74MB                                                                                                                  0.5s
 => => extracting sha256:820650aa2f5b8e795b04058fa617fb87ad3397f0fc2582400f9855b84f2d4401                                                                                                                       0.2s
 => => extracting sha256:4e0f3549117ae48d35d3cbea60bca5ef0d654723e8065274c93137c3f2b44a60                                                                                                                       0.8s
 => => extracting sha256:a5b6895e2863b5497adb5d68fe8e5241835c9624c9a31fd2b8532b08da56b638                                                                                                                       0.0s
 => => extracting sha256:629a421706977cd903f62bb22483a0fa6cba236a94d23543e616bf5c24516467                                                                                                                       0.3s
 => [app internal] load build context                                                                                                                                                                           0.1s
 => => transferring context: 10.25MB                                                                                                                                                                            0.1s
 => [worker internal] load build context                                                                                                                                                                        0.1s
 => => transferring context: 10.25MB                                                                                                                                                                            0.1s
 => [app main  2/10] WORKDIR /code                                                                                                                                                                              0.1s
 => [app main  3/10] RUN apk add build-base &&   apk add libffi-dev &&   pip install poetry==1.8.2                                                                                                             24.3s
 => [app main  4/10] COPY pyproject.toml /code/                                                                                                                                                                 0.0s
 => [app main  5/10] COPY poetry.lock /code/                                                                                                                                                                    0.0s
 => [app main  6/10] RUN poetry config virtualenvs.create false &&  poetry install --only main --no-interaction --no-ansi                                                                                       8.7s
 => [app main  7/10] COPY ./app /code/app                                                                                                                                                                       0.0s
 => [worker main  8/10] COPY ./static /code/static                                                                                                                                                              0.1s
 => [app main  9/10] COPY ./build/overfast-crontab /etc/crontabs/root                                                                                                                                           0.0s
 => [app main 10/10] RUN chmod +x /etc/crontabs/root                                                                                                                                                            0.2s
 => [worker] exporting to image                                                                                                                                                                                 1.3s
 => => exporting layers                                                                                                                                                                                         1.2s
 => => writing image sha256:097b9c11ac8d43b68212e99b3aaa27c6faaa9496a4a56c2dd17d1e8e7bf5b007                                                                                                                    0.0s
 => => naming to docker.io/library/overfast-api-worker                                                                                                                                                          0.0s
 => [app] exporting to image                                                                                                                                                                                    1.3s
 => => exporting layers                                                                                                                                                                                         1.2s
 => => writing image sha256:0d945663dff46a7417e3bee5a459b9bdbe73b8b0bf075392d2382d10e3253569                                                                                                                    0.0s
 => => naming to docker.io/library/overfast-api-app                                                                                                                                                             0.0s
 => [nginx internal] load build definition from Dockerfile                                                                                                                                                      0.0s
 => => transferring dockerfile: 284B                                                                                                                                                                            0.0s
 => [nginx internal] load metadata for docker.io/library/nginx:1.20.2-alpine                                                                                                                                    1.2s
 => [nginx internal] load .dockerignore                                                                                                                                                                         0.0s
 => => transferring context: 2B                                                                                                                                                                                 0.0s
 => [nginx 1/4] FROM docker.io/library/nginx:1.20.2-alpine@sha256:016789c9a2d021b2dcb5e1c724c75ab0a57cc4e8cd7aab7bb28e69fec7c8c4fc                                                                              1.0s
 => => resolve docker.io/library/nginx:1.20.2-alpine@sha256:016789c9a2d021b2dcb5e1c724c75ab0a57cc4e8cd7aab7bb28e69fec7c8c4fc                                                                                    0.0s
 => => sha256:455c02918c4592a9beeeae47df541266f3ea53ed573feb767e5e8ab8dcee146e 2.72MB / 2.72MB                                                                                                                  0.2s
 => => sha256:08b6b842dcd703db7772921ad226048f2f8b0bec2e1fcaaa55a59b66c4772db3 7.14MB / 7.14MB                                                                                                                  0.3s
 => => sha256:016789c9a2d021b2dcb5e1c724c75ab0a57cc4e8cd7aab7bb28e69fec7c8c4fc 1.65kB / 1.65kB                                                                                                                  0.0s
 => => sha256:1a946edcecbd2fd3121946a05e978c3a03c5d94c67d2376d3fbb62ed0c470d90 1.57kB / 1.57kB                                                                                                                  0.0s
 => => sha256:d6a1e2ab00f777c5cc735d3a1c78769cea2e2a220511c162907da2c6758e916f 8.76kB / 8.76kB                                                                                                                  0.0s
 => => sha256:d49e54c17cd83bb015a10bea1e05084d9630afc9a4feac1aa3ceb5ae647dd9d8 603B / 603B                                                                                                                      0.4s
 => => extracting sha256:455c02918c4592a9beeeae47df541266f3ea53ed573feb767e5e8ab8dcee146e                                                                                                                       0.1s
 => => sha256:0263029b2ae8863b72df3f821517b4ac5cfaff6cd482cca560636edee07d91a2 894B / 894B                                                                                                                      0.3s
 => => sha256:7a15a92d87b89cf5803fd585091af1cd8ba3216f8bb9639c7e7beb8358f88f4b 664B / 664B                                                                                                                      0.4s
 => => extracting sha256:08b6b842dcd703db7772921ad226048f2f8b0bec2e1fcaaa55a59b66c4772db3                                                                                                                       0.5s
 => => sha256:9fed390110deadba2d9f188f3d75b177f94ceababaf7d603ed64b30ad346ee0d 1.39kB / 1.39kB                                                                                                                  0.5s
 => => extracting sha256:d49e54c17cd83bb015a10bea1e05084d9630afc9a4feac1aa3ceb5ae647dd9d8                                                                                                                       0.0s
 => => extracting sha256:0263029b2ae8863b72df3f821517b4ac5cfaff6cd482cca560636edee07d91a2                                                                                                                       0.0s
 => => extracting sha256:7a15a92d87b89cf5803fd585091af1cd8ba3216f8bb9639c7e7beb8358f88f4b                                                                                                                       0.0s
 => => extracting sha256:9fed390110deadba2d9f188f3d75b177f94ceababaf7d603ed64b30ad346ee0d                                                                                                                       0.0s
 => [nginx internal] load build context                                                                                                                                                                         0.0s
 => => transferring context: 92.88kB                                                                                                                                                                            0.0s
 => [nginx 2/4] COPY nginx.conf /etc/nginx/nginx.conf                                                                                                                                                           0.0s
 => [nginx 3/4] COPY ngx_http_redis_module.so /usr/lib/nginx/modules/ngx_http_redis_module.so                                                                                                                   0.0s
 => [nginx 4/4] COPY overfast-api.conf /etc/nginx/conf.d/default.conf                                                                                                                                           0.0s
 => [nginx] exporting to image                                                                                                                                                                                  0.0s
 => => exporting layers                                                                                                                                                                                         0.0s
 => => writing image sha256:b17cb20d6e98fc6abf923c2c485464ed198eae699a116e9fdddc583ea98ba96b                                                                                                                    0.0s
 => => naming to docker.io/library/overfast-api-nginx                                                                                                                                                           0.0s
Stopping OverFastAPI and cleaning containers...
docker compose down -v --remove-orphans
WARN[0000] /home/dev/overfast-api/docker-compose.yml: `version` is obsolete 
Launching OverFastAPI (production mode)...
docker compose up -d
WARN[0000] /home/dev/overfast-api/docker-compose.yml: `version` is obsolete 
[+] Running 5/5
 ✔ Network overfast-api_default     Created                                                                                                                                                                     0.1s 
 ✔ Container overfast-api-redis-1   Healthy                                                                                                                                                                     4.6s 
 ✘ Container overfast-api-app-1     Error                                                                                                                                                                      19.6s 
 ✔ Container overfast-api-worker-1  Started                                                                                                                                                                     4.1s 
 ✔ Container overfast-api-nginx-1   Created                                                                                                                                                                     0.0s 
dependency failed to start: container overfast-api-app-1 is unhealthy
make: *** [Makefile:65: up] Error 1
```

**Expected behavior**
overfast-api runs well

**Screenshots**


**Additional context**
my server specs:
![image](https://github.com/TeKrop/overfast-api/assets/52338732/f106e4ce-749c-4092-a599-3244a52c91ee)

and docker version:
![image](https://github.com/TeKrop/overfast-api/assets/52338732/e1d11c1f-f0d9-4112-bc33-92e58a0d518d)

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -1,4 +1,3 @@
-version: "3.8"
 services:
   app:
     build:
@@ -12,7 +11,7 @@ services:
       redis:
         condition: service_healthy
     healthcheck:
-      test: ["CMD-SHELL", "wget -O /dev/null http://localhost:8080 || exit 1"]
+      test: ["CMD-SHELL", "wget --spider --quiet http://0.0.0.0:8080 || exit 1"]
       interval: 5s
       timeout: 5s
 

